test/smoke-fails/omptest-testsuite-emi/Makefile

```diff
@@ -0,0 +1,18 @@
+include ../../Makefile.defs
+
+TESTNAME     = omptest-testsuite-emi
+TESTSRC_MAIN = omptest_testsuite_emi.cpp
+TESTSRC_AUX  =
+TESTSRC_ALL  = $(TESTSRC_MAIN) $(TESTSRC_AUX)
+
+CLANG        ?= clang++
+OMP_BIN      = $(AOMP)/bin/$(CLANG)
+RUNENV       += OMPTEST_USE_OMPT_EMI=1
+RUNCMD       = ./$(TESTNAME)
+CC           = $(OMP_BIN) $(VERBOSE) -I$(AOMP_REPOS)/llvm-project/openmp/libomptarget/test/ompTest/include -L$(AOMP_REPOS)/build/openmp/libomptarget -lomptest
+#-ccc-print-phases
+#"-\#\#\#"
+
+include ../Makefile.rules
+run:
+	$(RUNENV) $(RUNCMD)
```

Comment:
I think this can be omitted, since the Makefile should construct this rule automatically.